### PR TITLE
Some phrasing changes, small fixes, todos.

### PR DESCRIPTION
**Pull request by amaurremi**

### CODE REVIEW

File: source/smart-contracts/general/develop-contracts.rst [hunk 1]
```diff
@@ -42,7 +42,7 @@ compiled to Wasm.
 To obtain correct exports, the `crate-type` attribute must be set to
 ``["cdylib", "rlib"]`` in the manifest file:
 
-.. code-block::
+.. code-block:: console
```
Comment (Jasagredo) [inline source/smart-contracts/general/develop-contracts.rst:45]: I'm not sure if this should be a console block or just a plain code-block.

File: source/smart-contracts/tutorials/first-contract.rst [hunk 2]
```diff
@@ -288,7 +288,7 @@ a terminal:
 It should run one test, and hopefully it succeeds.
 
 
-``receive``-functions
+receive functions
```
Comment (Jasagredo) [inline source/smart-contracts/tutorials/first-contract.rst:291]:
```suggestion
Receive functions
```

File: source/smart-contracts/general/develop-contracts.rst (hunk 1 above)
Comment (amaurremi) [reply source/smart-contracts/general/develop-contracts.rst]: Makes sense, changed it to `text` now.

**Comment by abizjak** [comment]
Can we get this merged ASAP. There are too many interdependent changes at the same time and we need to be wrapping things up (there are already conflicts).

**Review by limemloh — APPROVED** [review]

**Pull request merged**